apps/web/src/components/FormList.tsx

```diff
@@ -37,8 +39,8 @@ export const FormList = ({
         .map((formInstance) => ({
           ...formInstance,
           levenshteinDistance: distance(
-            searchQuery.toLowerCase(),
-            formInstance.name.toLowerCase(),
+            searchQuery.toLowerCase().slice(0, 10),
+            formInstance.name.toLowerCase().slice(0, 10),
           ),
         }))
         .sort((a, b) => a.levenshteinDistance - b.levenshteinDistance),
```

Comment:
Just noting here that we might want to do a .slice after this in case we have too many forms